table/queries/merged_reduced_scans.sql

```diff
@@ -84,23 +84,23 @@ CREATE TEMP FUNCTION ClassifyError(error STRING,
     WHEN REGEXP_CONTAINS(error, "malformed MIME") THEN "http/http.invalid"
 
     # Content verification failures
-    WHEN (source = "ECHO" AND ENDS_WITH(error, "EOF")) THEN "content/response_mismatch" # Echo
+    WHEN (source = "ECHO" AND ENDS_WITH(error, "EOF")) THEN "content/unexpected" # Echo
     # Hyperquack v1 errors
-    WHEN (error = "Incorrect echo response") THEN "content/response_mismatch" # Echo
-    WHEN (error = "Received response") THEN "content/response_mismatch" # Discard
-    WHEN (error = "Incorrect web response: status lines don't match") THEN "content/status_mismatch" # HTTP/S
-    WHEN (error = "Incorrect web response: bodies don't match") THEN "content/body_mismatch" # HTTP/S
-    WHEN (error = "Incorrect web response: certificates don't match") THEN "content/tls_mismatch" # HTTPS
-    WHEN (error = "Incorrect web response: cipher suites don't match") THEN "content/tls_mismatch" # HTTPS
-    WHEN (error = "Incorrect web response: TLS versions don't match") THEN "content/tls_mismatch" # HTTPS
+    WHEN (error = "Incorrect echo response") THEN "content/unexpected" # Echo
+    WHEN (error = "Received response") THEN "content/unexpected" # Discard
+    WHEN (error = "Incorrect web response: status lines don't match") THEN "content/unexpected_status" # HTTP/S
```

Comment:
Not for this PR, but it would be great to output the status number here. I don't know if that's available.

Reply:
agreed